mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
if you want but make it private

On Thu, Oct 22, 2015 at 11:25 AM, jaeilepp <notifications@github.com> wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/2551#discussion_r42727339>:
>
> > @@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
> >          # pick channels
> >          if picks is None:
> >              picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
> > -                               stim=False, eog=False, ecg=False,
> > +                               stim=False, eog=False, ecg=False, seeg=True,
> >                                 emg=False, exclude=[])
>
> How about a separate convenience function like pick_data_channels?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2551/files#r42727339>.
>